content/en/docs/concepts/security/securing-pods.md

```diff
@@ -0,0 +1,216 @@
+{{< /note >}}
+
+#### Applying to All Namespaces
+
+If you're just getting started with the Pod Security Standards, a suitable first step would be to configure all namespaces as `privileged` but set up audit annotations for a stricter level such as `baseline`:
+
+```shell
+kubectl label --overwrite ns --all \
+  pod-security.kubernetes.io/enforce=privileged \
+  pod-security.kubernetes.io/audit=baseline \
+  pod-security.kubernetes.io/warn=baseline
+```
+
+#### Applying to a Single Namespace
+
+You can update a specific namespace as well. This command adds the `enforce=restricted` policy to `my-existing-namespace`, pinning the restricted policy version to v1.22.
+
+```shell
+kubectl label --overwrite ns my-existing-namespace \
+  pod-security.kubernetes.io/enforce=restricted \
+  pod-security.kubernetes.io/enforce-version=v1.22
+```
+
+## Migrating from Pod Security Policies
+
+Migrating to the replacement policy from PodSecurityPolicies can be done effectively using a combination of dry-run and `audit` and `warn` modes, although this becomes harder if mutating PSPs are used).
+
+{{< note >}}
+**We are working on tooling to automate this migration.** In the meantime, this guide should give you an idea of the general process.
+{{< /note >}}
+
+- [Enable the Pod Security Standards feature gate.](/docs/concepts/security/securing-pods#enabling-the-alpha-feature)
+- **Temporarily set the default `enforce` mode to `privileged` for all Namespaces.** This can be done by [configuring the Admission Controller](/docs/concepts/security/securing-pods#configuring-the-admission-controller) or with [`kubectl label`](/docs/concepts/security/securing-pods#applying-to-all-namespaces).
+- **Eliminate mutating Pod Security Policies, if your cluster has any set up.**
+  - Clone all mutating PSPs into a non-mutating version.
+  - Update all ClusterRoles authorizing use of those mutating PSPs to also authorize use of the non-mutating variant.
+  - Watch for Pods using the mutating PSPs and work with code owners to migrate to valid, non-mutating resources.
+  - Delete mutating PSPs.
+- **Select a compatible policy level for each namespace.** Analyze existing resources in the namespace to drive this decision; strive for the `restricted` and `baseline` levels.
+  - Review the requirements of the different [Pod Security Standards](/docs/concepts/security/pod-security-standards).
+  - Evaluate the difference in privileges that would come from disabling the PSP controller.
+- **Apply the selected profiles in `warn` and `audit` mode.** This will give you an idea of how your Pods will respond to the new policies, without breaking existing workloads. Iterate on your [Pods' configuration](/docs/concepts/security/securing-pods#configuring-pods) until they are in compliance with the selected profiles.
+- Apply the profiles in enforce mode.
+- Disable PodSecurityPolicy!
+
+## FAQ
```

Comment:
This could be a What's Next heading.
```suggestion
## {{% heading "whatsnext" %}}

- Read the [Pod Security Standards](/docs/concepts/security/pod-security-standards/)
…
```

Reply:
Thoughts on where those FAQs should go?

Reply:
These aren't really FAQs (nobody is bombarding us with support, not yet). Let's try to mention the answers in the page another way, and also cover them in the post-release blog article about this new feature.